knative-operator/deploy/resources/kourier/download.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 KOURIER_VERSION=v0.3.10
-DOWNLOAD_URL=https://raw.githubusercontent.com/3scale/kourier/${KOURIER_VERSION}/deploy/kourier-knative.yaml
+DOWNLOAD_URL=https://raw.githubusercontent.com/openshift-knative/kourier/${KOURIER_VERSION}/deploy/kourier-knative.yaml
```

Comment:
Although I changed to download manifest from `openshift-knative/kourier`, actually it is not synced with Kourier upstream yet. So it does not have v0.3.10 yet.